zest/releaser/tests/postrelease.txt

```diff
@@ -40,7 +30,7 @@ testing hack in utils.py:
 
 Run the postrelease script:
 
-    >>> utils.test_answer_book.set_answers(['', ''])
+    >>> utils.test_answer_book.set_answers(['', '', 'n'])
```

Comment:
I've seen the extra `n` showing up a couple of times. Where's that coming from?
Perhaps "push to the server" that you didn't have with svn?

....

Ah wait, I see it below :-)